src/Core/Common/UseParallelizeImageRegion/Code.cxx

```diff
@@ -0,0 +1,116 @@
+/*=========================================================================
+ *
+ *  Copyright Insight Software Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#include "itkLogImageFilter.h"
+#include "itkRandomImageSource.h"
+#include "itkImageDuplicator.h"
+#include "itkImageRegionIterator.h"
+
+constexpr unsigned int Dimension = 2;
+using PixelType = unsigned int;
+using ImageType = itk::Image< PixelType, Dimension >;
+
+//calculate log(1+x), where x is pixel value, using LogImageFilter
+void log1x_LogImageFilter(ImageType::Pointer& image)
```

Comment:
Is there any particular reason to use underscores in function names here? This is not consistent with the ITK style :worried: